src/main/java/org/scify/jedai/progressivejoin/SetSimJoin.java

```diff
@@ -26,10 +27,9 @@
 public class SetSimJoin {
 
     private LinkedHashMap<String, ArrayList<Integer>> records;
-    private ArrayList<Object[]> results;

```

Comment:
I was a bit confused by the format of these object arrays so I thought I'd change it to a first class object. Originally I created
```
public class PairSimilairty (
    float similarity;
    int recordId1;
    int recordId2;
}
```
But then noticed it was pretty similar to the existing `Comparison` object so I opted to use that instead. If you find the usage of `Comparison` unclear, I'd be fine adding back a class like `PairSimilarity`.